GameOfLifeApp/Program.cs

```diff
@@ -1,2 +1,17 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using GameOfLifeApp;
+
+var data = new GameLogic();
+data.SetUpField();
+
+while (true)
+{
+    Console.Clear();
+    Console.SetCursorPosition(0, 0);
+    data.GetNextGeneration();
```

Comment:
CurrentField is In Display class.

Reply:
What i meant is, after you call `data.SetUpField();`, you never print these values, we never see the initial state of the field. You immediate go into while loop and in while loop, you call the `data.GetNextGeneration();` and then print it, so, we miss the very first iteration of the field.

Reply:
Displaying of the random field.

Reply:
Not quite, you dont need to call the `Display.ShowRandomField(data);` method in every while loop, only once, before the while loop as that is the only time, when it will display different value from your other `Display.ShowIteration(data);` method.

Reply:
Done.